src/components/toast/toast.style.ts

```diff
@@ -27,8 +28,9 @@ const StyledPortal = styled(Portal)<{
 
     ${align === "left" &&
     css`
-      left: 12%;
-      transform: translateX(-50%);
+      display: flex;
```

Comment:
Question: Do we need `flex` here?

Reply:
Good point, no I don't think there is a need for it here. I took it from the styles that were set for `align === "right"` and just flipped it for left, but I think it should be fine to remove `flex` from both left and right as it doesn't seem to affect the styling.